source/Table/Table.js

```diff
@@ -416,6 +416,7 @@ export default class Table extends React.PureComponent {
 
         <Grid
           {...this.props}
+          aria-readonly={null}
```

Comment:
Not allowed on [rowgroup](https://www.w3.org/TR/wai-aria-1.1/#rowgroup): https://www.w3.org/TR/html-aria/#allowed-aria-roles-states-and-properties